docs/plans/index.html

```
@@ -0,0 +1,461 @@
+<body>
+
+  <div class="gallery-header">
+    <h1>Plans Library</h1>
+    <p>1 plans &mdash; click any card to open it</p>
+  </div>
+
+  <div class="toolbar">
+    <input type="text" class="search-box" id="searchBox"
+           placeholder="Search by title..." aria-label="Search plans by title">
+    <div class="view-toggle">
+      <button class="view-btn active" id="gridBtn" title="Grid view" aria-label="Grid view" aria-pressed="true">&#9638;</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)